cdk/lib/index.ts

```diff
@@ -296,7 +296,7 @@ dpkg -i /tmp/package.deb`,
       autoMinorVersionUpgrade: true,
       backupRetention: Duration.days(10),
       engine: DatabaseInstanceEngine.postgres({
-        version: PostgresEngineVersion.VER_13_3,
+        version: PostgresEngineVersion.VER_13,
```

Comment:
I can't seem to make this comment on the correct line, but I'd strongly recommend setting `allowMajorVersionUpgrade` above to `false` too to avoid surprises (breaking changes etc.)

Reply:
Agreed – worth encoding this as a default in CDK? (Happy to raise a PR!)